web/src/components/organisms/Settings/Members/hooks.ts

```diff
@@ -59,6 +59,20 @@ export default ({ workspaceId }: Props) => {
     [searchUserQuery],
   );
 
+  const workspaceUserMembers = useMemo((): Member[] | undefined => {
```

Comment:
Sorry I thought I mentioned this in the first review but must have forgot to comment about this. Shouldn't this be either workspaceUsers OR workspaceMembers? Why workspaceUserMembers?

Reply:
@KaWaite , at first backend only had **members** in each workspace, then member became one of two possible types: either **userMembers** or **integrationMembers**.

![1](https://user-images.githubusercontent.com/78056580/194187934-10660a39-9624-4b12-9f21-c8bc526c4b43.png)

Reply:
@KaWaite , btw, I'll create another PR, where all members (old **Member**) become **UserMembers**.
The main focus here is integration right now.